doc/filters/format_currency.rst

```diff
@@ -1,7 +1,7 @@
 ``format_currency``
 ===================
 
-The ``format_currency`` filter formats a number as a currency:
+The ``format_currency`` filter formats a number (float) as a currency:
```

Comment:
No, passing an integer is totally valid.

* https://www.php.net/manual/fr/numberformatter.format.php
* https://github.com/twigphp/intl-extra/blob/8a953b1d8d8be4bef57508c4b825d508fcca6efe/IntlExtension.php#L335

Reply:
You're right, I changed it